frontend/src/pages/SessionForm/pages/__tests__/participants.js

```diff
@@ -253,14 +273,52 @@ describe('participants', () => {
       expect(virtuallyLabel).toBeVisible();
     });
 
+    it('shows read only mode correctly for ist visit selection', async () => {
+      const readOnlyFormValues = {
+        ...defaultFormValues,
+        pocComplete: true,
+        pocCompleteId: userId,
+        pocCompleteDate: todaysDate,
+        event: {
+          pocIds: [userId],
+        },
+        recipients: [
+          {
+            id: 1,
+            label: 'R1 R1 G1',
+          },
+        ],
+        deliveryMethod: 'hybrid',
```

Comment:
Do we need another test for the isHybrid branch?

Reply:
"Hybrid" wasn't added in this PR, but there's one in this file.